HumanControls/DashPage.cpp

```diff
@@ -10,28 +10,50 @@ void DashPage::paint(DisplayController &display, bool isActivated, JsonElement &
 {
     display.clear();
 
-    if (object["hCtrl"]["conn"].asString()[0] == 'A')
+    if (canActivate() && isActivated)
     {
-        display.printRegion(0, 0, object["status"].asString());
+        for (int i = 0; i < strlen(object["err"].asString()); i++)
+        {
+            if (i < 16)
```

Comment:
I recently shortened the error field to 16 chars, so it will always be one line from now on.